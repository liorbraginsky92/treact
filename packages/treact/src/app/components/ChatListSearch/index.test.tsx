import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { ChatListSearch } from './'

it('ChatListSearch renders correctly', () => {
  const tree = renderer.create(
    <ChatListSearch />,
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
