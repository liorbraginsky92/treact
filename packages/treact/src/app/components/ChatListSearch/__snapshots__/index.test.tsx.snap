// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatListSearch renders correctly 1`] = `
<div
  className="css-c5c4qt"
>
  <input
    className="css-wdr6gu"
    placeholder="Search"
    type="text"
  />
  <div
    className="css-171m698"
  />
</div>
`;
